src/utils/utils.js

```diff
@@ -230,20 +230,13 @@ export async function resolveBorrowerCheck(agencyId, context) {
  * @returns {string}
  */
 export async function getInfomediaAccessStatus(context) {
-  if (!context?.smaug?.user?.id) {
-    return "USER_NOT_LOGGED_IN";
-  }
+  const user = context?.user;
 
-  let userInfo;
-  try {
-    userInfo = await context.datasources.getLoader("userinfo").load({
-      accessToken: context.accessToken,
-    });
-  } catch (e) {
+  if (!user?.userId) {
```

Comment:
Hvorfor er det ikke alle stedet at dette tjek foretages? Er det fordi man skal være logget ind for at bruge infomedia?

Reply:
Ja det er en funktion som henter en access status ud for infomedia (getInfomediaAccessStatus)

hvis der ikke er et userId er man ikke loggedIn og vi ved ikke om man har et adgangsgivende bibliotek.

Reply:
Okay. En kommentar ville være nice